tests/integration/test_refreshable_mv/test.py

```diff
@@ -312,47 +313,49 @@ def test_refreshable_mv_in_read_only_node_no_ddl(started_cluster, cleanup):
     reading_node.replace_in_config(
         "/etc/clickhouse-server/users.d/users.xml", "<allow_ddl>1", "<allow_ddl>0"
     )
-    reading_node.query("SYSTEM RELOAD CONFIG")
+    try:
+        reading_node.query("SYSTEM RELOAD CONFIG")
 
-    assert "DDL queries are prohibited" in reading_node.query_and_get_error(
-        "create table foo (x Int64) engine = Null;"
-    )
+        assert "DDL queries are prohibited" in reading_node.query_and_get_error(
+            "create table foo (x Int64) engine = Null;"
+        )
 
-    node1.query(
-        "create table re.foo (x Int64) engine = ReplicatedMergeTree order by x;"
-    )
+        node1.query(
+            "create table re.foo (x Int64) engine = ReplicatedMergeTree order by x;"
+        )
 
-    node1.query("system stop view sync")
+        node1.query("system stop view sync")
 
-    # this MV will be replicated to read_only node
-    node1.query(
-        "create materialized view re.a refresh every 1 second (x Int64) engine ReplicatedMergeTree order by x as select number*10 as x from numbers(2)",
-    )
+        # this MV will be replicated to read_only node
+        node1.query(
+            "create materialized view re.a refresh every 1 second (x Int64) engine ReplicatedMergeTree order by x as select number*10 as x from numbers(2)",
+        )
 
-    reading_node.query("system refresh view re.a")
+        reading_node.query("system sync database replica re")
```

Comment:
(This was missing, so `system refresh view re.a` sometimes failed because the table doesn't exist yet.)